src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java

```diff
@@ -127,6 +135,20 @@ private static Imp modifyImp(Imp imp) {
         return imp;
     }
 
+    private void validateVideoParams(Video video) {
+        final Integer w = video.getW();
```

Comment:
It is to cryptic to see. May be it will be better to add universal method like `positiveOrNull(..)` for each param.
Then in condition below we can compare with NULL only.